src/SFA.DAS.FindEpao.Api/Controllers/CoursesController.cs

```diff
@@ -107,15 +108,17 @@ public async Task<IActionResult> CourseEpao(int id, string epaoId)
                 var model = new GetCourseEpaoResponse
                 {
                     Course = queryResult.Course,
+                    StandardVersions = queryResult.StandardVersions
+                        .Select(item => (StandardVersions)item),
                     Epao = queryResult.Epao,
                     CourseEpaosCount = queryResult.CourseEpaosCount,
                     EffectiveFrom = queryResult.EffectiveFrom,
                     EpaoDeliveryAreas = queryResult.EpaoDeliveryAreas
-                        .Select(area => (EpaoDeliveryArea) area),
+                        .Select(area => (EpaoDeliveryArea)area),
                     DeliveryAreas = queryResult.DeliveryAreas
-                        .Select(item => (GetDeliveryAreaListItem) item),
+                        .Select(item => (GetDeliveryAreaListItem)item),
                     AllCourses = queryResult.AllCourses
-                        .Select(item => (GetCourseListItem)item)
+                        .Select(item => (GetAllCoursesListItem)item)
```

Comment:
![image](https://user-images.githubusercontent.com/54770494/143274904-acafa175-6302-49b0-8ef6-07bbdcc73e16.png)
AllCourses should have StandardVersions as an array, not a string. https://skillsfundingagency.atlassian.net/wiki/spaces/NDL/pages/3495657555/Find+an+EPAO+-+SV

Reply:
changed to array